packages/kit/src/runtime/client/fetcher.js

```diff
@@ -19,7 +19,7 @@ if (DEV && BROWSER) {
 	let can_inspect_stack_trace = false;
 
 	// detect whether async stack traces work
-	const check_stack_trace = async () => {
+	const check_stack_trace = () => {
```

Comment:
this is there for a reason! without it, we're not actually checking anything

```suggestion
	const check_stack_trace = async () => {
```

Reply:
perhaps we could avoid the ignore by writing it as this?
```
	const check_stack_trace = () => {
		const stack = /** @type {string} */ (new Error().stack);
		can_inspect_stack_trace = stack.includes('check_stack_trace');
	};

	// detect whether async stack traces work
	Promise.resolve(check_stack_trace());
```

Reply:
That completely misses the point, which is to find out whether stack traces cross async boundaries in the current browser.

Reply:
I've honestly got not idea what that means. Is this some kind of browser compatibility thing where that happens only in some browsers, but not others?